client/src/components/App.jsx

```diff
@@ -17,10 +18,11 @@ class App extends React.Component {
       const date = new Date;
       return (
         <div>
-          <Navbar type={this.state.data[1].type} location={this.state.data[1].location}
-            reviews={this.state.data[1].reviewnum} stars={this.state.data[1].reviewsummary}
-            year={this.state.data[1][date.getFullYear() + 1]} click={() => this.showModal()}/>
-          <Modal show={this.state.modal} hide={() => this.closeModal()}/>
+          <Navbar type={this.state.data[this.state.listingId].type} location={this.state.data[this.state.listingId].location}
```

Comment:
I think you should make variables for type, location, etc. for readability

Reply:
Will do.

Reply:
Updated.